packages/kit/types/config.d.ts

```diff
@@ -80,6 +80,7 @@ export interface Config {
 		vite?: ViteConfig | (() => ViteConfig);
 	};
 	preprocess?: any;
+	[key: string]: any;
```

Comment:
This specific code would cause typescript errors without it:

![image](https://user-images.githubusercontent.com/34116392/130362002-61383f40-5587-40e6-984f-00db09858184.png)

Plus if we do use that exported type in a Typescript file (in userland code), we wouldn't be able to define the extra keys. But I don't really see much use-case for that, especially that we're using it in jsdoc mainly.

So I guess it'll better to `@ts-ignore` that specific case to keep the codebase's type safety at the meantime then.